prime.cs

```diff
@@ -0,0 +1,45 @@
+﻿
+//find out if the entered number is primary or not primary. 
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace prime
+{
+    class Prime
+    {
+        static void Main(string[] args)
+        {
+            int number = Convert.ToInt32(Console.ReadLine());
+            int i = 2;
+            if (number == 1 || number == 0)
+            {
+                Console.WriteLine("Primary");
+            }
+            else
+            {
+                while (i < number)
+                {
+                    if (number % i == 0)
+                    {
+                        Console.WriteLine("Not Primary");
+                        break;
+                    }
+                    else
+                    {
+                        i++;
```

Comment:
Good practice to increment the counter after using it. And the best is putting it outside of the `if` `else` inside the `while` because this counter is actually for the `while`.

You can keep like this if you want, but just wanted to mention that to you.